include/flox/core/util.hh

```diff
@@ -351,6 +351,28 @@ bool isUInt( std::string_view str );
 bool hasPrefix( std::string_view prefix, std::string_view str );
 
 
+/* -------------------------------------------------------------------------- */
+
+/** @brief trim from start ( in place ). */
+std::string & ltrim( std::string & str );
+
+/** @brief trim from end ( in place ). */
+std::string & rtrim( std::string & str );
+
+/** @brief trim from both ends ( in place ). */
+std::string & trim( std::string & str );
+
+
+/** @brief trim from start ( copying ). */
+[[nodiscard]] std::string ltrim_copy( std::string_view str );
+
+/** @brief trim from end ( copying ). */
+[[nodiscard]] std::string rtrim_copy( std::string_view str );
+
+/** @brief trim from both ends ( copying ). */
+[[nodiscard]] std::string trim_copy( std::string_view str );
```

Comment:
Should we use boost instead?

Reply:
We could, I just dropped these in to avoid digging around boost and adding it as a direct dependency.
This works for now.